### ISSUE TITLE
fix: aria-describedby in fast-tooltip should be read by screen readers

### ISSUE THREAD

**Issue by brookdozer** [microsoft/fast#5002]
# 🐛 Bug Report

When a tooltip anchor has an `aria-describedby` attribute set to the ID of the tooltip, the text of the tooltip should be read by screen readers, but that is not happening.

## 💻 Repro or Code Sample

1. Navigate to https://typescript-gsanag.stackblitz.io/
2. Turn on screen reader
3. Note that screen reader doesn't read the text of the tooltip, despite the button having `aria-describedby`

## 🤔 Expected Behavior

The [documentation](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/tooltip/tooltip.spec.md#implementation) seems to imply that tooltip will work for AT to describe its anchor. 

## 😯 Current Behavior

No text is read aloud, despite the `aria-describedby` attribute.

## 🔦 Context

The tooltip works as as additional description for fully sighted users but users with visual impairments do not get the benefit of this extra description.

Not the same, but related to feature request #5001.

## 🌍 Your Environment

Windows

* OS & Device: Windows 10 - Narrator, MacOS 11.4 - VoiceOver
* Browser: Microsoft Edge, Mozilla FireFox
* Version [e.g. 1.8.0]

**Comment by chrisdholt** [comment]
@litteredwitherrors are you able to dig into this and see what might be going on?

**Comment by scomea** [comment]
That's because the tooltip contents don't actually exist in the DOM until it is invoked.

**Comment by litteredwitherrors** [comment]
The issue does not exclusively pertain to `fast-tooltip` or `fast-button`, but any element used to describe a `fast-button` using `aria-describedby`. The issue is stemming components that use `delegatesFocus`. Both Narrator and VoiceOver are not reading descriptors of components which delegate focus to elements in its own shadow DOM. 

This issue should be closed in favor of: https://github.com/microsoft/fast/issues/5215

**Comment by chrisdholt** [comment]
Closing this in favor of #5215
